content/operator/ltg/index.de.md

```diff
@@ -0,0 +1,94 @@
+---
+draft: false
+title: "LTG-Link"
+country:
+  - "lithuania"
+  - "latvia"
+operator: "ltg"
+---
+
+Die LTG ist die staatliche Eisenbahngesellschaft der Republik Litauen und betreibt mit ihrer Tochtergesellschaft LTG-Link alle Verbindungen im Personenverkehr auf dem litauischen Schienennetz sowie ins Ausland.
+
+## Zusammenfassung
+
+- FIP 50 und FIP Freifahrtscheine werden akzeptiert.
```

Comment:
Aber nicht auf der Strecke von Vilnius nach Tallinn. Wobei ich online zumindest kostenlose Reservierung bis nach Riga buchen kann, auch wenn die "Zug" international ist, also hängt es nicht vom Zug ab. Ab Riga gibt es keine vergünstigten Tickets mehr, z. B. Vilnius - Zemitani kostet dann Normalpreis. Ticketsplitting ergibt hier unter Umständen Sinn, da der kostenfreie Abschnitt Vilnius - Riga nicht berücksichtig wird, wenn man durchbucht.

Reply:
Ja LTG-Link fährt ab Riga "im Auftrag" für Vivi (Latvian Railways) (https://www.seat61.com/trains-and-routes/vilnius-riga-tallinn-by-train.htm)

Ich schreibe das nochmal deutlicher, dass man bis Riga fahren kann.

Reply:
```suggestion
- FIP 50 und FIP Freifahrtscheine werden auf allen Routen außer zwischen Riga und Tallinn akzeptiert
```

Würde es expliziter schreiben, wie in dem Beispiel. Wenn ich im Navigator nach der Verbindung suche, sehe ich als Betreiber nur LTG (ich weiß, die Information ist nicht immer korrekt, aber dennoch...), würde mir die LTG Seite durchlesen und lese, dass FIP akzeptiert wird. Nur in einem "Persönlichen Tipp" könnte man erahnen, dass es nicht weiter nach Tallinn gilt. In den Zug wird zwar niemand damit steigen, weil keine kostenlose Reservierung ausgetellt wird, aber zur Reiseplanung kann das schon ungeschickt sein.